nifi-mock/src/main/java/org/apache/nifi/util/MockControllerServiceInitializationContext.java

```diff
@@ -68,4 +68,6 @@ public ComponentLog getLogger() {
     public StateManager getStateManager() {
         return stateManager;
     }
+
+
```

Comment:
nit-pick: this is the only change to this file (line at the end). Can it be removed?